packages/insert-menu/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "@sanity/insert-menu",
+  "version": "0.0.0",
+  "description": "",
+  "keywords": [],
+  "homepage": "https://github.com/sanity-io/visual-editing/tree/main/packages/insert-menu#readme",
+  "bugs": {
+    "url": "https://github.com/sanity-io/visual-editing/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/sanity-io/visual-editing.git",
+    "directory": "packages/insert-menu"
+  },
+  "license": "MIT",
+  "author": "Sanity.io <hello@sanity.io>",
+  "sideEffects": false,
+  "type": "module",
+  "exports": {
+    ".": {
+      "source": "./src/index.ts",
+      "require": "./dist/index.cjs",
+      "default": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "build": "pkg build --strict --clean --check",
+    "lint": "eslint .",
+    "ts:check": "tsc --build"
```

```suggestion
    "ts:check": "tsc --noEmit"
```
Oops we don't use incremental tsc builds anymore, we use turborepo instead 🙌

---

Fixed: 588a3971f53d1e12073b5f9a8617d7d819bc3d83